src/languages/es.js

```diff
@@ -1194,6 +1194,29 @@ export default {
     },
   },
 
+  coordinator_participant_history: {
```

Comment:
We already have these in es.js want to make a copy of some of them?
  coordinator: {
    heading: "Progreso diario de los participantes",
    participant_info: "Información del participante",
    name: "Nombre",
    today: "Hoy",
    medication: "Control de medicación",
    side_effects: "Efectos adversos o indeseables",
    photo: "Resultados del test de orina",
    actions: "Acción",
    contact: "Contacto",
    notes: "Notas",
    treatment: "Tratamiento",
    adherence: "TODO % Adherence",
    start_date: "Fecha de inicio del tratamiento",